src/components/Sum.jsx

```diff
@@ -1 +1,4 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum
+  = ({ a = 0, b = 0 }) => <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>;
```

Comment:
Instead of default values = 0, propTypes should be used:
`Sum.propTypes = {
a: PropTypes.number,
b: PropTypes.number,
};`

Reply:
explain pls

Reply:
```suggestion
export function Sum({ a, b }) {
  return (
    <p>
      {`Sum of ${a} and ${b} is ${a + b}`}
    </p>
  );
}

Sum.defaultProps = {
  a: 0,
  b: 0,
};
```

Reply:
why do i have to use it? is there something in the checklist or what?